app/services/rapid_ready_for_decision/hypertension_pdf_generator.rb

```diff
@@ -33,174 +45,25 @@ def medications?
       @medications.any?
     end
 
-    def add_intro
+    def patient_name
       names = @patient.with_indifferent_access
```

Comment:
:TIL: https://apidock.com/rails/ActiveSupport/HashWithIndifferentAccess

Reply:
Any theories as to why? Seems it's always keyed with symbols...

Reply:
It's possible that `patient` is keyed by strings (or maybe a mix of strings and symbols), and this class uses symbols exclusively.